ingest/api/ingestapi.py

```diff
@@ -55,7 +55,8 @@ def set_token(self, token):
     # TODO think of a better way how to manage tokens in this module, allowing clients to unset token for now
     def unset_token(self):
         self.token = None
-        del self.headers['Authorization']
+        if 'Authorization' in self.headers:
```

Comment:
A one liner replacement:
```suggestion
        self.headers.pop('Authorization', None)
```

Reply:
>The pop() method returns:
If key is found - removed/popped element from the dictionary
If key is not found - value specified as the second argument (default)
If the key is not found and default argument is not specified - KeyError exception is raised

This will work, but I like the 2 lines better. It's not that bad. I think it shows the intention more. Pop usage is more for using the removed value. My intention is to just delete the key. If I read the code and I would have to understand the specifics of the pop method to understand that I'm specifying a second param so that it won't raise a key error .

Reply:
on second thought, i think it wouldn't be bad to use pop either. Ignore my reason.